source/integration-guide/advanced.rst

```diff
@@ -327,3 +327,68 @@ To enable the purchase history tooltip add the ``tooltipEnabled`` option to the
         tooltipEnabled: true,
         tooltipStyle: "light"
     });
+
+
+Add to cart
+^^^^^^^^^^^
+You can enable an add to cart button directly inside the Virtusize widget.
+
+While it is not required, we recommend that you provide Virtusize with a list
+of **available sizes** when you enable add to cart. If you do so, the add to
```

Comment:
Can you hyperlink "available sizes" to the documentation for setAvailableSizes(sizesList) in http://developers.virtusize.com/integration-guide/advanced.html ? (or elsewhere, if there is a better place to link to)